engine/src/objdefs.h

```diff
@@ -214,6 +214,7 @@ along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
 #define F_PLAY_SELECTION        (1UL << 20)
 #define F_SHOW_SELECTION        (1UL << 21)
 #define F_SHOW_VOLUME           (1UL << 22)
+#define F_DONTUSEQT             (1UL << 23)
```

Comment:
As this is a runtime only property - there's no need to allocate a F_ flag for it.